src/Application/src/CalculatorApplication.cpp

```diff
@@ -1,47 +1,49 @@
 ////////////////////////////////////////////////////////////////////////////
 /**
- *  @file   Calculator.cpp
+ *  @file   CalculatorApplication.cpp
  *  @date   Fri April 16 2021
  *  @brief  These are the functions that are used by main.cpp
  */
 ////////////////////////////////////////////////////////////////////////////
 
-#include "../inc/Calculator.h"
-#include "../inc/Messages.h"
+#include "CalculatorStrings.h"
+#include "CalculatorApplication.h"
+#include <iostream>
```

Comment:
have system headers before library headers and have a space between them. They should also be in alphabetical order in their respective categories. E.g., for this file:

```
#include <iostream>
#include <limits>

#include "CalculatorApplication.h"
#include "CalculatorStrings.h"
```

update all other files to match